### PR TITLE
Python: add finally and with statements

### PR DESCRIPTION
**Pull request by outcoldman**
https://docs.python.org/2/tutorial/errors.html

### CODE REVIEW

File: python.html.markdown [hunk 1]
```diff
@@ -379,7 +379,13 @@ except (TypeError, NameError):
     pass    # Multiple exceptions can be handled together, if required.
 else:   # Optional clause to the try/except block. Must follow all except blocks
     print "All good!"   # Runs only if the code in try raises no exceptions
+finally: #  Execute under all circumstances
+    print "We can clean up resources here"
 
+# Instead of try/finally to cleanup resources you can use with
```
Comment (geoffliu) [inline python.html.markdown]: Instead of "with", could you say "a with statement"? Or enclose "with" in backticks.

**Comment by geoffliu** [comment]
Also while we're at it, could you update the Python3 page as well? Thanks!

File: python.html.markdown (hunk 1 above)
Comment (outcoldman) [inline python.html.markdown]: fixed!

**Comment by outcoldman** [comment]
@geoffliu of course, but I'm too lazy to enlist this repo, so created a separate pull request using GitHub UI

**Comment by geoffliu** [comment]
Thanks!

**Pull request merged**